ckanext/unaids/tests/test_logic.py

```diff
@@ -22,6 +24,8 @@
     ], ids=[
         "validates if all upload fields correct",
         "validates if no upload fields present",
+        "validates if all fields are present but empty",
+        "fails if some fields missing and others set to empty string",
```

Comment:
I am not sure we really want this? i.e. If some fields are missing and all others present but empty, should this pass validation or not? I have not allowed this here since it is an edge case that would involve a much more substantial change to the original validator logic.

Reply:
I think it's ok behaviour which disables partial input (you should either provide all or none blob storage metadata).